presto-native-execution/pom.xml

```diff
@@ -152,4 +152,45 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludes>
+                                <!-- Exclude tests against Glue from build -->
+                                <exclude>**/TestHiveQueriesGlueMetastoreUsingJSON.java</exclude>
+                                <exclude>**/TestHiveQueriesGlueMetastoreUsingThrift.java</exclude>
+                            </excludes>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>test-hive-glue</id>
```

Comment:
Just for my knowledge why is this separate profile needed ? It seems like its exactly the same as the default profile that is excluding the Glue tests.

Reply:
Glue tests require AWS credentials so we limit them to only Glue-related tests. This is similar to how Hive tests are set up. You can see how we define environment variables when "test-hive-glue" profile is enabled in YAML files.